Sources/AccountSDKIOSWeb/Lib/API/SchibstedAccountAPIResponses.swift

```diff
@@ -1,48 +1,46 @@
 import Foundation
 
 public struct UserProfileResponse: Codable, Equatable {
-    public var uuid: String? = nil
-    public var userId: String? = nil
-    public var status: Int? = nil
-    public var email: String? = nil
-    internal var emailVerified: StringOrIgnore? = nil
-    public var emails: [Email]? = nil
-    public var phoneNumber: String? = nil
-    internal var phoneNumberVerified: StringOrIgnore? = nil
-    public var phoneNumbers: [PhoneNumber]? = nil
-    public var displayName: String? = nil
-    public var name: Name? = nil
-    public var addresses: [String: Address]? = nil
-    public var gender: String? = nil
+    public var uuid: String?
+    public var userId: String?
+    public var status: Int?
+    public var email: String?
+    internal var emailVerified: StringOrIgnore?
+    public var emails: [Email]?
+    public var phoneNumber: String?
+    internal var phoneNumberVerified: StringOrIgnore?
+    public var phoneNumbers: [PhoneNumber]?
+    public var displayName: String?
+    public var name: Name?
+    public var addresses: [String: Address]?
+    public var gender: String?
     internal var birthday: String?
-    public var accounts: [String: Account]? = nil
-    public var merchants: [Int]? = nil
-    public var published: String? = nil
-    public var verified: String? = nil
-    public var updated: String? = nil
-    public var passwordChanged: String? = nil
-    public var lastAuthenticated: String? = nil
-    public var lastLoggedIn: String? = nil
-    public var locale: String? = nil
-    public var utcOffset: String? = nil
+    public var accounts: [String: Account]?
+    public var merchants: [Int]?
+    public var published: String?
+    public var verified: String?
+    public var updated: String?
+    public var passwordChanged: String?
+    public var lastAuthenticated: String?
+    public var lastLoggedIn: String?
+    public var locale: String?
+    public var utcOffset: String?
```

Comment:
Not related to this PR but should all of these properties really be optional?

Reply:
I have no idea 🤔 👀 I will check that and create a ticket if its necessary

Reply:
I think we could make use of structs for API responses and use non-optionals for everything that is required to return from the API. Makes it easier to work without guard checks

Reply:
I agree -> https://github.schibsted.io/spt-identity/user-access-team/issues/95